bgpd/bgp_vty.c

```diff
@@ -2677,6 +2677,19 @@ static int peer_remote_as_vty(struct vty *vty, const char *peer_str,
 	return bgp_vty_return(vty, ret);
 }
 
+DEFUN (bgp_default_auto_shutdown,
+       bgp_default_auto_shutdown_cmd,
+       "[no] bgp default auto-shutdown",
```

Comment:
Maybe drop the auto and just `[no] bgp default shutdown`?  Just because the neighbor equivalent is `shutdown`

Reply:
makes sense to me